crates/forge/src/cmd/lint.rs

```diff
@@ -100,6 +108,12 @@ impl LintArgs {
             None => config.lint.severity.clone(),
         };
 
+        // Override default fail_on level config with user-defined severity
```

Comment:
can simplify `self.fail_on.unwrap_or(config.lint.fail_on)` and move below inline